pkgs/development/libraries/openssl/default.nix

```
@@ -154,7 +153,7 @@ let
     postInstall =
     (if static then ''
       # OPENSSLDIR has a reference to self
-      ${removeReferencesTo}/bin/remove-references-to -t $out $out/lib/*.a
+      ${buildPackages.removeReferencesTo}/bin/remove-references-to -t $out $out/lib/*.a
```

```suggestion
      remove-references-to -t $out $out/lib/*.a
```
Why can't we add it to nativeBuildInputs?

---

#192084

---

Sure, either way is cool with me. A couple of lines down `${buildPackages....}` was used for something else so I just did it this way to keep it consistent.

---

In a replace we cannot really use nativeBuildInputs.